PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.core/src/com/microsoft/azuretools/core/ui/startup/HDInsightHelpDlg.java

```diff
@@ -20,11 +20,15 @@
 
 import com.microsoft.azuretools.core.Activator;
 import com.microsoft.azuretools.core.utils.Messages;
+import com.microsoft.azuretools.core.utils.PluginUtil;
 
 
 public class HDInsightHelpDlg extends Dialog {
 	private boolean isShowTips = true;
 	
+	private static String scalaPluginMarketplaceURL = "http://marketplace.eclipse.org/marketplace-client-intro?mpc_install=421";
```

Comment:
Yes, it is only for Scala plugin check currently.

---
In reply to: [132092815](https://github.com/Microsoft/azure-tools-for-java/pull/845#discussion_r132092815) [](ancestors = 132092815)